cpp_tests/feature/one_hot_encoder_test.cc

```diff
@@ -14,79 +14,153 @@
  * limitations under the License.
  */
 
-#include "fstream"
-#include <cstdlib>
-#include <string>
-#include <sys/stat.h>
-
 #include "clink/cpp_tests/test_util.h"
 #include "clink/feature/one_hot_encoder.h"
-#include "nlohmann/json.hpp"
 #include "gtest/gtest.h"
 
-namespace tfrt {
+namespace clink {
+
 namespace {
 
-TEST(OneHotEncoderTest, Param) {
-  clink::OneHotEncoderModel model;
-  model.setDropLast(false);
-  EXPECT_EQ(model.getDropLast(), false);
-  model.setDropLast(true);
-  EXPECT_EQ(model.getDropLast(), true);
+class OneHotEncoderTest : public testing::Test {
+protected:
+  static void SetUpTestSuite() {
+    if (host_context == nullptr) {
```

Comment:
Would it be simpler to `assert(host_context == nullptr)` here? Same for `mlir_context`.